### DISCUSSION TITLE
Notifications - Webkit and Email

### DISCUSSION THREAD

**Discussion by isaaczaak** [ourzora/nouns-builder#226] [Ideas]
https://warpcast.com/0xdesigner/0xdddc12

**Comment by neokry** [comment]
I would prioritize email over webkit first for a few reasons
- email services have been around for years whereas webkit notifs are pretty new / could have some issues 
- webkit notifs are specific to users that have the website saved to their homescreen so theres more friction for receiving them
- would need to integrate with both ios and android web notifications
